frontend/src/modules/controllers/ValueModal.tsx

```diff
@@ -1,15 +1,19 @@
 /**
- * @summary Re-usable wrapper components to display & control value
- * Currently unused, setValueContent is used in MultiStepPopup
+ * ValueModal
  *
+ * @component A container for setting the value of modal settings in multistep popup.
```

Comment:
Can we make this description clearer and more specific and also provide more context? e.g. it's only the content shown inside a modal for adjusting the values of parameters in the wizard, but the content for adjusting the values of alarm limits is in AlarmModal.tsx, etc.